components/ContactList.js

```diff
@@ -26,19 +26,44 @@ import ContactItem from "../components/ContactItem";
 // Erin Larson – (542) 321-3456
 // .....
 
-export default function ContactList() {
+export default function ContactList({ listFavorites }) {
   const dispatch = useDispatch();
-  const contacts = useSelector(getContacts);
+  const contacts = listFavorites
+    ? useSelector(getFavorites)
+    : useSelector(getContacts);
```

Comment:
might be better to pass these different lists in as a prop from pages/index.js to make this component more reusable